Directory.Build.props

```diff
@@ -2,23 +2,17 @@
 
   <PropertyGroup>
     <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
-    <ApplicationInsightsPackageVersion>2.21.0</ApplicationInsightsPackageVersion>
```

Comment:
I am removing these properties, as we can now control which packages are updated together via the Renovate config instead of this workaround. We will continue to use MSBuild properties for versions that vary by other factors like `TargetFramework`